leaf-ui/js/displayAnalysis.js

```diff
@@ -133,58 +129,38 @@ function adjustSliderWidth(maxValue){
  * @param {AnalysisResult} currentAnalysis
  *  an AnalysisResult object that contains data about the analysis that the back end performed
  */
+// TODO: Reimplement with new backbone models
 function updateSliderValues(sliderValue, currentAnalysis){
 
-    analysisResult.selectedTimePoint = sliderValue;
+    // analysisResult.selectedTimePoint = sliderValue;
 
-    var value = sliderValue;
-    $('#sliderValue').text(value);
-    sliderObject.sliderValueElement.innerHTML = value + "|" + currentAnalysis.timePointPath[value];
-    // Update the analysisRequest current state.
-    analysisRequest.currentState = sliderObject.sliderValueElement.innerHTML;
-    currentAnalysis.elementList.forEach(element => 
-        updateNodeValues(element.id, element.status[value]));
+    // var value = sliderValue;
```

Comment:
Yes, I think it is a good protocol to comment out the code that needs to be updated with the new backbone models, but leave the functions call there with a TODO. :)